### PR TITLE
Make end to end example using colab

### PR DESCRIPTION
**Pull request by yupbank**
Since colab support query bigquery and provide free gpu/tpu.
Add a notebook to do the following steps 
-  Query big query to get a dataset
- Use tensorflow hub to reuse pre-trained word embedding
- Train a keras model end to end with a string input and probability output.

Preview on colab https://colab.research.google.com/drive/1KXUqoNLX9-2vEs1FB30jhj-ofjunz-Je

### CODE REVIEW

**Comment by jlewi** [comment]
@yupbank Is there anything about this notebook that makes it specific to colab as opposed to jupyter?

How is this notebook different from the existing notebooks?
https://github.com/machine-learning-apps/Issue-Label-Bot/tree/master/notebooks

/assign @hamelsmu

**Comment by yupbank** [comment]
mostly colab offers free gpu/tpu, and have a bunch of python libs pre-installed. 
to perform an end to end workflow, using big query requires authentication.

**Comment by hamelsmu** [comment]
@yupbank I think this notebook will confuse matters in this project.  I would rather add the colab badge to the existing notebooks already in place.

**Comment by hamelsmu** [comment]
I'm going to go ahead and close this notebook in favor of https://github.com/machine-learning-apps/Issue-Label-Bot/pull/51/files.  🙇

**Comment by hamelsmu** [comment]
@yupbank I'm happy to reopen this PR if you want to make the case and write more documentation on how this notebook is different than the other notebook(s) and what the purpose of this notebook is vs. the other notebook, including creating a README in the /notebooks folder.  

Before you do that work, might be worthwhile to discuss your goals with the PR further.  Thank you for getting involved with the project!
